notifications_api_common/kanalen.py

```diff
@@ -3,27 +3,35 @@
 """
 
 from collections import defaultdict
-from typing import Dict, Tuple
+from typing import Dict, Tuple, Union
 
 from django.core.exceptions import FieldDoesNotExist, ImproperlyConfigured
-from django.db.models import Model
-from django.db.models.base import ModelBase
+from django.db.models import Field, Model
+
+from rest_framework.request import Request
 
 KANAAL_REGISTRY = set()
 
 
 class Kanaal:
-    def __init__(self, label: str, main_resource: ModelBase, kenmerken: Tuple = None):
+    def __init__(
+        self,
+        label: str,
+        main_resource: Model,
+        kenmerken: Union[Tuple, None] = None,
+        extra_kwargs: Union[Dict, None] = None,
```

Comment:
Am I correct that you use `extra_kwargs` only to retrieve help texts? Could you please add docstring or more restrictive typehint for it, so it would be more clear how to use this parameter

Reply:
I've added a more restrictive typehint for it